soh/src/code/padmgr.c

```diff
@@ -269,7 +269,7 @@ void PadMgr_ProcessInputs(PadMgr* padMgr) {
         input->press.stick_y += (s8)(input->cur.stick_y - input->prev.stick_y);
     }
 
-    controllerCallback.rumble = padMgr->rumbleEnable[0] > 0 ? 1 : 0;
+    controllerCallback.rumble = (CVar_GetS32("gRumbleEnabled", 0) != 0) && (padMgr->rumbleEnable[0] > 0) ? 1 : 0;
```

Comment:
nit:  you don't need the != 0.  For that matter, you don't need the ? 1 : 0 either, even though it was weirdly originally there.